src/LSST/M1M3/SS/Controllers/PositionController.cpp

```diff
@@ -288,7 +291,11 @@ bool PositionController::moveHardpoint(int32_t steps, int hpIndex) {
     return true;
 }
 
+#ifdef WITH_SAL_KAFKA
+bool PositionController::move(std::vector<int> steps) {
+#else
 bool PositionController::move(int32_t *steps) {
```

Comment:
This is the same as with ForceController::applyOffsetForces. Drop #ifdef here, construct const vector in command handling. Or call .data() to get pointer to the first int in the vector.